python3/tests/test_raygunprovider.py

```diff
@@ -134,43 +137,49 @@ def create_dummy_message(self):
         msg.set_exception_details(errorMessage)
         return msg.build()
 
+    def test_message_with_error(self):
+        msg = self.create_dummy_message()
+        self.sender.on_grouping_key(self.the_callback_with_error)
+        msg =self.sender._transform_message(msg)
+        self.assertEqual(msg.get_details()["groupingKey"], "Exception: None")
+
```

Comment:
this test ensures that the `_transform_message` retuns a properly formatted `RaygunMessage` and doesn't break into a `dict` like before

Reply:
not necessary anymore due to changes